markdown/AstroChem/nelson.md

```diff
@@ -11,151 +11,173 @@ using ODEInterface, ODEInterfaceDiffEq
+    # 8: O
+    du[8] = n_H * (-2e-10) * u[7] * u[8] + 
+            n_H * 1.6e-9 * u[10] * u[5] - 
+            n_H * 8e-10 * u[2] * u[8] + 
+            5e-10 * Go * exp(-1.7 * Av) * u[9] + 
+            1e-10 * Go * exp(-3 * Av) * u[10] * shield
+
+    # 9: OHx
+    du[9] = n_H * (-1e-9) * u[9] * u[12] + 
+            n_H * 8e-10 * u[2] * u[8] - 
+            n_H * 5.8e-12 * (T^0.5) * u[9] * u[6] - 
+            5e-10 * Go * exp(-1.7 * Av) * u[9]
+
+    # 10: CO
+    du[10] = n_H * (3.3e-5 * u[11] * u[3]) / T + 
+             n_H * 2e-10 * u[7] * u[8] - 
+             n_H * 1.7e-9 * u[10] * u[2] - 
+             n_H * 1.6e-9 * u[10] * u[5] + 
+             n_H * 5.8e-12 * (T^0.5) * u[9] * u[6] - 
+             1e-10 * Go * exp(-3 * Av) * u[10] + 
+             1.5e-10 * Go * exp(-2.5 * Av) * u[11] * shield
+
+    # 11: HCO+
+    du[11] = n_H * (-3.3e-5 * u[11] * u[3]) / T + 
+             n_H * 1e-9 * u[9] * u[12] + 
+             n_H * 1.7e-9 * u[10] * u[2] - 
+             1.5e-10 * Go * exp(-2.5 * Av) * u[11]
+
+    # 12: C+
+    du[12] = n_H * (-1.4e-10 * u[3] * u[12]) / (T^0.61) - 
+             n_H * 4e-16 * u[1] * u[12] - 
+             n_H * 1e-9 * u[9] * u[12] + 
+             n_H * 1.6e-9 * u[10] * u[5] + 
+             3e-10 * Go * exp(-3 * Av) * u[6]
+
+    # 13: M+
+    du[13] = n_H * (-3.8e-10 * u[13] * u[3]) / (T^0.65) + 
+             n_H * 2e-9 * u[2] * u[14] +
+             2.0e-10 * Go * exp(-1.9 * Av) * u[14]
+
+    # 14: M
+    du[14] = n_H * (3.8e-10 * u[13] * u[3]) / (T^0.65) - 
+             n_H * 2e-9 * u[2] * u[14] -
+             2.0e-10 * Go * exp(-1.9 * Av) * u[14]
 
 end
 
-
-u0 = [0.5 ;    # 1:  H2   yep?
-	9.059e-9 ; # 2:  H3+  yep
-	2.0e-4 ;   # 3:  e    yep
-	0.1 ;              # 4:  He  SEE lines 535 NL99
-	7.866e-7 ; # 5:  He+  yep? should be 2.622e-5
-	0.0 ;      # 6:  C    yep
-	0.0 ;      # 7:  CHx  yep
-	0.0004 ;   # 8:  O    yep
-	0.0 ;      # 9:  OHx  yep
-	0.0 ;      # 10: CO   yep
-	0.0 ;      # 11: HCO+ yep
-	0.0002 ;   # 12: C+   yep
-	2.0e-7 ;   # 13: M+   yep
-	2.0e-7 ]   # 14: M    yep
-
-
-tspan = (0.0, 30 * 3.16e10) # ~30 thousand yrs
-
-prob = ODEProblem(Nelson_ODE, u0, tspan)
+# Set the Timespan, Parameters, and Initial Conditions
+seconds_per_year = 3600 * 24 * 365
+tspan = (0.0, 30000 * seconds_per_year) # ~30 thousand yrs
+
+params = (10,  # T
+          2,   # Av
+          1.7, # Go
+          611, # n_H
+          1)   # shield
+
+u0 = [0.5,      # 1:  H2
+      9.059e-9, # 2:  H3+
+      2.0e-4,   # 3:  e
+      0.1,      # 4:  He
+      7.866e-7, # 5:  He+
+      0.0,      # 6:  C
+      0.0,      # 7:  CHx
+      0.0004,   # 8:  O
+      0.0,      # 9:  OHx
+      0.0,      # 10: CO
+      0.0,      # 11: HCO+
+      0.0002,   # 12: C+
+      2.0e-7,   # 13: M+
+      2.0e-7]   # 14: M
+
+
+prob = ODEProblem(Nelson!, u0, tspan, params)
 refsol = solve(prob, Vern9(), abstol=1e-14, reltol=1e-14)
+sol2 = solve(prob, FBDF())
+sol3 = solve(prob, QNDF())
+sol4 = solve(prob, CVODE_BDF())
```

Comment:
Is there a reason for using the high tolerances here? That's almost certainly the reason why they are different from the reference.